build_web_compilers/lib/src/dev_compiler_stack_trace/stack_trace_mapper.dart

```diff
@@ -78,8 +78,7 @@ class LazyMapping extends Mapping {
       var rawMap = _provider(uri);
       if (rawMap != null) {
         var strMap = rawMap is String ? rawMap : _stringify(rawMap);
-        var mapping = parse(strMap) as SingleMapping;
-        mapping
+        var mapping = parse(strMap) as SingleMapping
```

Comment:
cases like this where we were already using cascades, but on the variable rather than within the assignment, are more idiomatic after applying the lint